test/cppsuite/tests/run.cpp

```diff
@@ -234,6 +240,11 @@ main(int argc, char *argv[])
             for (const auto &test_name : all_tests)
                 std::cout << "\t" << test_name << std::endl;
             return 0;
+        } else if (option == "-H") {
```

Comment:
Wish we could have A->Z in this function too but might be too much work to re sort everything, up to you.

Reply:
Nah not in this change, theres a bunch of option cleanup work we could do which I think should be separate. Such as removing all the local variables and just using the test_args structure.